packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -245,13 +253,15 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
 
     // Render SXA Rendering Variant
     if (renderingDefinition.params?.FieldNames) {
-      return componentFactory(
-        renderingDefinition.componentName,
-        renderingDefinition.params.FieldNames
-      );
+      return withErrorBoundary(
+        this.props.errorComponent,
+        this.props.errorBoundaryComponent
+      )(componentFactory(renderingDefinition.componentName, renderingDefinition.params.FieldNames));
     }
 
-    return componentFactory(renderingDefinition.componentName);
+    return withErrorBoundary(this.props.errorComponent)(
```

Comment:
Looks like this is missing the `this.props.errorBoundaryComponent` parameter?

Reply:
errorComponent is what will be shown in case of error
erroBoundaryComponent is custom error boundary - if anyone decides to use it